bin/improver-nbhood-land-and-sea

```diff
@@ -174,34 +174,48 @@ def main():
         radius_or_radii = args.radii_by_lead_time[0].split(",")
         lead_times = args.radii_by_lead_time[1].split(",")
 
-    if masking_coordinate is not None:
-        result_land = ApplyNeighbourhoodProcessingWithAMask(
-            masking_coordinate, radius_or_radii, lead_times=lead_times,
-            ens_factor=args.ens_factor, sum_or_fraction=args.sum_or_fraction,
-            re_mask=False).process(cube, mask)
-    else:
-        result_land = NeighbourhoodProcessing(
+    if land_only.data.max() > 0.0:
```

Comment:
Can we put comments in to say that which section is dealing with land and which with sea, i.e. what each top level if statement is for.